### ISSUE TITLE
Internationalization API is not available on some mobile browsers

### ISSUE THREAD

**Issue by tiltec** [yunity/karrot-frontend#748]
We received some errors today from Chrome 51 on Android 7. It seems that it doesn't support Intl, although it should:
https://caniuse.com/#feat=internationalization

When digging further, it seems to be a common problem on Samsung phones, which can be worked around by providing a polyfill:
https://github.com/Financial-Times/polyfill-service/issues/509

I'm not a huge fan of bloating our codebase with polyfills, but maybe a polyfill service could help?
https://polyfill.io/

https://sentry.io/foodsaving-worldwide/karrot/issues/403966409/

```
ReferenceError: Intl is not defined
  at _localizeDateTime (./node_modules/vue-i18n/dist/vue-i18n.esm.js:1333:1)
  at _d (./node_modules/vue-i18n/dist/vue-i18n.esm.js:1351:1)
  at apply (./node_modules/vue-i18n/dist/vue-i18n.esm.js:1391:1)
  at $d (./node_modules/vue-i18n/dist/vue-i18n.esm.js:194:1)
  at call (js/0.463cada66b1a8d3b51c0.js:16005:19)
...
(11 additional frame(s) were not displayed)
```

**Comment by nicksellen** [comment]
13kb gzipped https://bundlephobia.com/result?p=intl@1.2.5 which would be nice to avoid, although using polyfill.io means an extra roundtrip.

... but then there are the locales to think about too. but depending on what vue-i18n uses of Intl, maybe they aren't needed.

I think we could just not support locales for those browsers anyway (here is the handling https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_negotiation).

If I understand this plugin correctly though, it will not make a request unless it needs to: https://www.npmjs.com/package/webpack-polyfill-injector#use-polyfillio-cdn. I will try it :)

Otherwise this should do it...
https://polyfill.io/v2/polyfill.js?features=Intl&unknown=polyfill

**Comment by nicksellen** [comment]
Almost... gets a new error now:

> No locale data has been provided for this object yet.

https://sentry.io/foodsaving-worldwide/karrot/issues/406102245/?referrer=slack

**Comment by JakeChampion** [comment]
To load locale data from polyfill you need to tell it which locales you want. E.G. https://polyfill.io/v2/polyfill.js?features=Intl,Intl.~locale.en-GB&unknown=polyfill would load the Intl API and the en-GB locale data if the browser does not already have it.

**Comment by nicksellen** [comment]
It was resolved #767 using webpack dynamic imports which turned out to be a nice solution.
